yabi/yabi/settings.py

```diff
@@ -595,25 +587,26 @@
     'loggers': {
         '': {
             'handlers': ['console', 'file'],
-            'level': 'INFO',
+            'level': 'DEBUG' if DEBUG else 'INFO',
+            'propagate': True
         },
         'django': {
-            'handlers': ['console', 'django_file'],
+            'handlers': ['django_file', 'mail_admins'],
```

Comment:
Also applies to all the additions of `'main_admins` handlers below. Instead of adding 'mail_admin' filter to all the entries, just add it to the root logger entry above, to
  - have less repetition
  - ensure that all ERRORS are mailed not just the ones in the loggers we list

The `'mail_admins'` handler is filtered